target/linux/ath79/dts/ar9344_huawei_ap6010dn.dts

```diff
@@ -0,0 +1,242 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "ar9344.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	model = "Huawei AP6010DN";
+	compatible = "huawei,ap6010dn", "qca,ar9344";
+
+	chosen {
+		bootargs = "console=ttyS0,9600n8";
+	};
+
+	aliases {
+		led-boot = &led_function_green;
+		led-failsafe = &led_function_red;
+		led-running = &led_function_green;
+		led-upgrade = &led_function_red;
+	};
```

Comment:
`label-mac-device = &eth0`

Then remove the current reference in the shell script and check with this simple script if the returned mac-address matches the one on the label:

https://github.com/freifunk-gluon/gluon/blob/main/package/gluon-core/files/lib/gluon/label_mac.sh

Reply:
Thanks for the hint how to check.